run-terraform.sh

```diff
@@ -1,18 +1,21 @@
 #!/bin/bash
 
-cd credentials
-. ./credentials.sh
-cd ..
-
-cd run-dir
-rm *.tf
-cd ..
+if [ ! -d "run-dir" ]; then
```

Comment:
Add a brief comment on what this block is doing.

Reply:
Done